server/www/components/singleValue/singleValue.js

```diff
@@ -0,0 +1,68 @@
+//var resizingFont = false;// tells whether or nor we have already planned a onContainerResized()
+
+
+Vue.component('single-value', {
+name: 'single-value',
+props: {
+    widget: {type: Object, required: true},
+},
+computed: {
+    telem() { return this.widget.series[0].name; },
+    value() { return this.widget.singlevalue; },
+    unit() { return this.widget.series[0].unit; },
+    fillColor() { return this.widget.series[0].options.fill; },
+    strokeColor() { return this.widget.series[0].options.stroke; }
+},
+methods: {
+    onContainerResized()
+    {
+        if (this.$refs.telem_responsive_text == undefined)
+            return;
+        
+        this.$refs.telem_responsive_text.triggerTextResize();
+        this.$refs.value_responsive_text.triggerTextResize();
+        this.$refs.unit_responsive_text.triggerTextResize();
+    }
+},
+mounted() {
+    
+    const resizeObserverForSingleValue = new ResizeObserver((entries) => {
+        this.onContainerResized();
+    });
+      
+    resizeObserverForSingleValue.observe(document.getElementById('single-value-container-id'));
```

Comment:
When multiple single-value-widgets are displayed, they all have the same ids (which is not recommended!)